bin/kn-apply

```diff
@@ -3,13 +3,13 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
-host_cloud=$(grep -w "provider" config.tfvars |
-  cut -d "=" -f 2- |
-  awk -F\" '{print $(NF-1)}')
+# Read config file into variable as as json
+kn_config=$(json2hcl -reverse <config.tfvars)
 
-image_name=$(grep -w "boot_image" config.tfvars |
-  cut -d "=" -f 2- |
-  awk -F\" '{print $(NF-1)}')
+# Set variables from json-config
+host_cloud=$(jq -r '.provider' <<<"$kn_config")
+image_name=$(jq -r '.boot_image' <<<"$kn_config")
+skip_image_import=$(jq -r '.skip_image_import' <<<"$kn_config")
```

Comment:
Where do you set this var?

Reply:
Maybe this should be in the docs as well.

Reply:
Skip image import is not set yet, but it is used in a plugin config, where only Terraform part of KubeNow is used and a vanilla image is used.

Reply:
The other providers can use an alternative image just specifying the name (together with skip_image_import=true), Only Azure need alternative method in terraform scripts.